### DISCUSSION TITLE
New Project AirSim release date

### DISCUSSION THREAD

**Discussion by ITstollman** [microsoft/AirSim#4791] [General]
Does someone know when will the New Project AirSim will be realesd 
and will be open for the public to use?

I know that Microsoft are saying it will be released in the folliwing year, but maybe someone knows a more spesific date?
Does someone know a way for me to descover that issue?

Thank you
Itamar

**Comment by xxEoD2242** [comment]
Hey @ITstollman,

Not sure about Project AirSim. I've looked a lot as well. If you are interested in something now, we have developed the [SWARM Simulation Platform](https://www.swarmsim.io) and the [SWARM Developer Version](https://www.swarmsim.io/overview/developer) to help develop and deploy robotic systems. Feel free to reach out to me directly as well: tylerfedrizzi@codex-labs-llc.com.

We also are continuing the open-source project and improving it at [Colosseum](https://github.com/CodexLabsLLC/Colosseum)

**Reply by ITstollman** [reply]
Hi, Thanks for the respons. 

I prefer not to start working on the project with a tool that I know I will need to convert from. 

If you hear anything about the release date I will appreciate a share.

Thanks, 
Itamar

**Reply by xxEoD2242** [reply]
No worries.

Just to follow up, our platform is fully supported and available now. We offer similar features to what Microsoft has proposed but focus on single and multi agent systems beyond just aerial vehicles. And it's available today 😁

**Reply by ITstollman** [reply]
Thanks for letting me know. 

My project is focused on a single earial vehicle. 
If you have some advanced features over AirSim on a single earial vehicle - I would to hear about it.

**Reply by xxEoD2242** [reply]
Sure! Would you mind sending me an email so we can correspond in private?

**Reply by ITstollman** [reply]
Sure, in a moment :)

**Comment by valxv** [comment]
It will never be released. Microsoft shut it down at the end of 2023.
https://www.gadgets360.com/cryptocurrency/news/microsoft-airsim-shut-metaverse-project-focus-artificial-intelligence-ai-fires-staff-4511889

**Reply by AlexGisi** [reply]
IAMAI has begun to publish their version of the project: https://github.com/iamaisim/ProjectAirSim

**Comment by ITstollman** [comment]
Thanks!
Wonder how you remembered me

Itamar Stollman
